solidity/dashboard/src/components/LiquidityRewardCard.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react"
+import DoubleIcon from "./DoubleIcon"
+import * as Icons from "./Icons"
+import { SubmitButton } from "./Button"
+import Card from "./Card"
+
+const LiquidityRewardCard = ({ title, MainIcon, SecondaryIcon, viewPoolLink }) => {
+  return (
+    <Card className={"tile"}>
+      <div className={"liquidity__card-title-section"}>
+        <DoubleIcon
+          MainIcon={MainIcon}
+          SecondaryIcon={SecondaryIcon}
+          className={`liquidity__double-icon-container`}
+        />
+        <h2 className={"h2--alt text-grey-70"}>{title}</h2>
+      </div>
+      <div className={"liquidity-card-subtitle-section"}>
+        <span className="text-grey-40">Uniswap Pool&nbsp;</span>
+        <a
+          href={viewPoolLink}
+          className="arrow-link text-small"
```

Comment:
Unnecessary `arrow-link` class. We handle external links via [selector](https://github.com/keep-network/keep-core/blob/master/solidity/dashboard/src/css/typography.less#L105).

Reply:
Fixed in https://github.com/keep-network/keep-core/pull/2216/commits/0cd07235983324ca719889e5738b09bda0e4dc16